src/Sarif/Processors/Log/SarifLogActions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
+// license. See LICENSE file in the project root for full license information. 
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis.Sarif.Processors
+{
+    // Mapping Actions
+    public enum SarifLogAction
+    {
+        None = 0,
+        RebaseUri,
+        MakeUrisAbsolute,
+        Merge,
+        // Future work...
+        Sort,
+        MakeDeterministic,
```

Comment:
>MakeDeterministic [](start = 8, length = 17)

other actions, embed file contents, remove file contents, populate files table, provide file hashes.

note that sorting is implied by making the file deterministic, but not vice versa. Might need 'StripNondeterministicPathInformation', and then 'MakeDeterministic' == Sort | StripNondeterministicPathInformation